journal-writings/original/19-epistle.adoc

```diff
@@ -423,9 +423,7 @@ and that nothing of a defiling nature, tending to discord and wars, may remain a
 
 May each of us query with ourselves, have the treasures I possess,
 been gathered in that wisdom which is from above, so far as has appeared to me?
-
```

Comment:
Let's restore this line.

Reply:
repeat after me... "I promise to not change any paragraphs... I promise to not change any paragraphs.... I...." i'm not sure why I'm not following what you're trying to get me to restore.... but I'm not! if what I've done isn't right, take another whack at it, ok?